.pipelines/ci/templates/build-powertoys-steps.yml

```diff
@@ -169,8 +190,9 @@ steps:
     vsVersion: 17.0
     platform: '$(BuildPlatform)'
     configuration: '$(BuildConfiguration)'
-    msbuildArgs: /t:PowerToysBootstrapper ${{ parameters.additionalBuildArguments }} /p:PerUser=true
+    msbuildArgs: /t:PowerToysBootstrapper ${{ parameters.additionalBuildArguments }} /p:PerUser=true -bl:$(Build.ArtifactStagingDirectory)\logs\PowerToysSetup-PowerToysBootstrapper-PerUser.binlog -ds:false
```

Comment:
Q: What does `-ds:false` do?

Reply:
Turns off the detailed summary that comes along with the binlog. Without this it'll spew a whole bunch of perf data to the console, which if you wanted you could just get from the binlog (and you're not getting today anyway).